README.md

```diff
@@ -254,68 +253,69 @@ docker exec -it \
+Create two directories called `[custom]-worker` and `[custom]-nginx` in the `build` directory.
 
-    ```bash
-    cd frappe_docker
-    mkdir ./build/[custom]-worker ./build/[custom]-nginx
-    ```
+```shell
+cd frappe_docker
+mkdir ./build/[custom]-worker ./build/[custom]-nginx
+```
 
-2. Create a `Dockerfile` in `./build/[custom]-worker` with the following content:
+Create a `Dockerfile` in `./build/[custom]-worker` with the following content:
 
-    ```Dockerfile
-    FROM frappe/erpnext-worker:edge
+```Dockerfile
+FROM frappe/erpnext-worker:edge
 
-    RUN install_app [custom] https://github.com/[username]/[custom] [branch]
-    # Only add the branch if you are using a specific tag or branch.
-    ```
+RUN install_app [custom] https://github.com/[username]/[custom] [branch]
+# Only add the branch if you are using a specific tag or branch.
+```
 
-3. Create a `Dockerfile` in `./build/[custom]-nginx` with the following content:
+Create a `Dockerfile` in `./build/[custom]-nginx` with the following content:
 
-    ```Dockerfile
-    FROM bitnami/node:12-prod
+```Dockerfile
+FROM bitnami/node:12-prod
 
-    COPY build/[custom]-nginx/install_app.sh /install_app
+COPY build/[custom]-nginx/install_app.sh /install_app
 
-    RUN /install_app [custom] https://github.com/[username]/[custom]
+RUN /install_app [custom] https://github.com/[username]/[custom]
 
-    FROM frappe/erpnext-nginx:edge
+FROM frappe/erpnext-nginx:edge
 
-    COPY --from=0 /home/frappe/frappe-bench/sites/ /var/www/html/
-    COPY --from=0 /rsync /rsync
-    RUN echo -n "\n[custom]" >> /var/www/html/apps.txt
+COPY --from=0 /home/frappe/frappe-bench/sites/ /var/www/html/
+COPY --from=0 /rsync /rsync
+RUN echo -n "\n[custom]" >> /var/www/html/apps.txt
 
-    VOLUME [ "/assets" ]
+VOLUME [ "/assets" ]
 
-    ENTRYPOINT ["/docker-entrypoint.sh"]
-    CMD ["nginx", "-g", "daemon off;"]
-    ```
+ENTRYPOINT ["/docker-entrypoint.sh"]
+CMD ["nginx", "-g", "daemon off;"]
+```
 
-4. Copy over the `install_app.sh` file from `./build/erpnext-nginx`
+Copy over the `install_app.sh` file from `./build/erpnext-nginx`
 
-    ```bash
-    cp ./build/erpnext-nginx/install.sh ./build/[custom]-nginx
-    ```
+```shell
+cp ./build/erpnext-nginx/install.sh ./build/[custom]-nginx
+```
 
-5. Open up `./installation/docker-compose-custom.yml` and replace all instances of `[app]` with the name of your app.
+Open up `./installation/docker-compose-custom.yml` and replace all instances of `[app]` with the name of your app.
 
-    ```bash
-    sed -i "s#\[app\]#[custom]#" ./installation/docker-compose-custom.yml
-    ```
+```shell
+sed -i "s#\[app\]#[custom]#" ./installation/docker-compose-custom.yml
+```
 
-6. Install like usuall, except that when you set the `INSTALL_APPS` variable set it to `erpnext,[custom]`.
+Install like usual, except that when you set the `INSTALL_APPS` variable to `erpnext,[custom]`.
 
 ## Troubleshoot
```

Comment:
we should move this to wiki and just place a reference link here?

https://github.com/frappe/frappe_docker/wiki/Troubleshoot

Reply:
It is better to avoid using the Wiki is possible, wiki are always left obsolete and changes are not tracked properly

Reply:
We should perhaps move from wiki to readme (or other documentation file if readme becomes too big) and disable the wiki

Reply:
Troubleshoot are contextual. I don't mind a `troubleshoot.md` somewhere.

Recipes are user driven and need to be in wiki

I added How to get webhooks working (on linux) if anyone is just using frappe/erpnext as API.